### PR TITLE
Replace TagID with VisualID in UI / Start Transaction With VisualID

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/app/pages/charging-stations/charging-station-start-transaction/charging-stations-start-transaction-dialog-component.ts [hunk 1]
```diff
@@ -57,8 +58,9 @@ export class ChargingStationsStartTransactionDialogComponent implements OnInit {
     // Set
     this.title = data.title;
     this.chargeBoxID = data.chargeBoxID;
+    this.siteID = data.siteID;
     this.loggedUser = centralServerService.getLoggedUser();
-    this.canListUsers = this.authorizationService.canListUsers();
+    this.canListUsers = this.authorizationService.isSiteAdmin(this.siteID);
```
Comment (LucasBrazi06) [inline src/app/pages/charging-stations/charging-station-start-transaction/charging-stations-start-transaction-dialog-component.ts:63]: This should be sent back by the backend when retrieving the default tag/car...

File: src/app/services/authorization.service.ts [hunk 2]
```diff
@@ -235,6 +235,9 @@ export class AuthorizationService {
 
   public canStopTransaction(siteArea: SiteArea, badgeID: string) {
     if (this.canAccess(Entity.CHARGING_STATION, Action.REMOTE_STOP_TRANSACTION)) {
+      if (!!this.loggedUser && !!this.loggedUser.tagIDs && this.loggedUser.tagIDs.includes(badgeID)) {
```
Comment (LucasBrazi06) [inline src/app/services/authorization.service.ts:238]: I removed this check for a reason before.
Check the git history.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Double check my comments

File: src/app/pages/charging-stations/charging-station-start-transaction/charging-stations-start-transaction-dialog-component.ts (hunk 1 above)
Comment (mahdibenromdhane) [reply src/app/pages/charging-stations/charging-station-start-transaction/charging-stations-start-transaction-dialog-component.ts]: Done

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**